CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.1.1] - 2017-11-01
```

Comment:
On most open source source projects as there is no guarantee when a project will reviewed, accepted, released it is best to leave versioning to the maintainers.

Reply:
Also based on using `Changed` vs `Fixed` it would be a minor rather than a patch as we follow semver.